packages/blocks/src/api/validation.js

```diff
@@ -397,6 +397,32 @@ function getHTMLTokens( html ) {
 	return null;
 }
 
+/**
+ * Returns true if the next HTML tag closes the current tag, false otherwise. Note this check does not
+ * affect the array of tokens
```

Comment:
The note here makes me wonder rather than leaving this as a potential point of confusion: Do we even need to pass the full array, or could we just pass `nextToken` as `expectedTokens[ 0 ]`?